api/handlers/space_test.go

```diff
@@ -40,11 +46,25 @@ var _ = Describe("Space", func() {
 			OrganizationGUID: "the-org-guid",
 		}, nil)
 
+		requestValidator = new(fake.RequestValidator)
+		serviceOfferingRepo = new(fake.CFServiceOfferingRepository)
+		serviceBrokerRepo = new(fake.CFServiceBrokerRepository)
+		servicePlanRepo = new(fake.CFServicePlanRepository)
+		orgRepo = new(fake.CFOrgRepository)
+
 		apiHandler = handlers.NewSpace(
 			*serverURL,
 			spaceRepo,
+			orgRepo,
 			routeRepo,
 			requestValidator,
+			relationships.NewResourseRelationshipsRepo(
```

Comment:
We could pass nil in place of all repos that we know are not needed for include resolution and this way save some lines in the `var` section and int the `BeforeEach`. This is a minor detail, so just keep in mind for future PRs, no need to waste more time on this one right now